fhir-install/docker/copy-dependencies-db2-migration.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+###############################################################################
+# (C) Copyright IBM Corp. 2016, 2020
```

Comment:
```suggestion
# (C) Copyright IBM Corp. 2020
```

2020

Reply:
its based on a script that was written in 2016, so i left the copyright from that

Reply:
I'm OK marking as resolved.